packages/postgres-database/src/simcore_postgres_database/migration/versions/61fa093c21bb_adding_product_name_to_wallets.py

```diff
@@ -0,0 +1,34 @@
+"""adding product_name to wallets
+
+Revision ID: 61fa093c21bb
+Revises: ae72826e75fc
+Create Date: 2023-09-20 14:42:10.661569+00:00
+
+"""
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "61fa093c21bb"
+down_revision = "ae72826e75fc"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column("wallets", sa.Column("product_name", sa.String(), nullable=True))
+    op.execute(
+        sa.DDL("UPDATE wallets SET product_name = s4l WHERE product_name IS NULL")
```

Comment:
so everything s4l if undefined until now! why not osparc. NOTE that `osparc` will always exist in the database since it is in the migration scripts as a default product

SEE https://github.com/ITISFoundation/osparc-simcore/blob/59c34d86136ad0d4a2f584f2afc013ae32921ab7/packages/postgres-database/src/simcore_postgres_database/migration/versions/350103a7efbd_modified_products_table.py#L37